frontend/src/components/6.Swiping/SwipingPage.js

```diff
@@ -83,42 +84,66 @@ function SwipingPage(props) {
   }
 
   /**
- * @param {*} timer new restaurant details
- * @return {void}
- */
+   * @param {*} timer new restaurant details
+   * @return {void}
+   */
   function getNewCard(timer) {
-    setTime(socketContext.countdown/1000);
+    setTime(socketContext.countdown / 1000);
     try {
       setDecided(false);
       CardData.shift();
       if (CardData[0] !== undefined) {
         setData(CardData[0]);
       }
-    } catch (error) {
-    }
-  };
+    } catch (error) {}
+  }
 
   return (
     <>
       <h1 className='Title'> yumble</h1>
       <h1 className='TimeCounter'> Remaining time: {time}s</h1>
       <div className='MakeCentre'>
-        <button
-          className='YesOrNoButton'
-          id='YesButton'
-          onClick={clickedYes}
-        >
-          Keen!
-        </button>
-        <button
-          className='YesOrNoButton'
-          id='NoButton'
-          onClick={clickedNo}
-        >
-          Nope!
-        </button>
-
-        <SwipeCard data={Data} ></SwipeCard>
+        <Container style={{ marginTop: '2em' }}>
+          <Row lg={12} className='justify-content-md-center'>
+            <Col
+              style={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+              }}
```

Comment:
Do we need to extract all the styling into CSS files or is inline styling ok?

Reply:
It's only being used for those buttons so I'm fine with leaving them as inline for now. However if we'll be using it for other components, I'll put it in a CSS file.